src/Tribe/Customizer/Global_Elements.php

```diff
@@ -476,49 +478,68 @@ public function setup() {
 	public function register_settings( WP_Customize_Section $section, WP_Customize_Manager $manager ) {
 		$customizer = Tribe__Customizer::instance();
 
+		// Add an heading that is a Control only in name: it does not, actulally, control or save any setting.
+		$manager->add_control( new Heading(
```

Comment:
:no_entry_sign: Opening parenthesis of a multi-line function call must be the last content on the line

<sub>hash: d34fef40b1affa17769f222bcea53d2a</sub>